cms/static/js/factories/videos_index.js

```diff
@@ -51,7 +52,10 @@ define([
                                 videoHandlerUrl: videoHandlerUrl,
                                 collection: updatedCollection,
                                 encodingsDownloadUrl: encodingsDownloadUrl,
-                                videoImageSettings: videoImageSettings
+                                videoImageSettings: videoImageSettings,
+                                transcriptAvailableLanguages: transcriptAvailableLanguages,
+                                videoSupportedFileFormats: videoSupportedFileFormats,
+                                isVideoTranscriptEnabled: isVideoTranscriptEnabled
```

Comment:
Why we are passing the last two?

Reply:
`isVideoTranscriptEnabled` is passed to check the availability of transcript feature, while `videoSupportedFileFormats` is needed at one place to make human-readable transcript filenames.